.github/workflows/tests.yml

```diff
@@ -92,6 +92,11 @@ jobs:
         run: |
           python -c "import $MODULE_NAME" -Werror
 
+      - name: Test CLI
+        run: |
+          python -m weasel --help
+        if: matrix.python_version == '3.9'
```

Comment:
This doesn't require anything extra/time-consuming like downloading models, so I think it makes sense to run it across the board.

```suggestion
```